### PR TITLE
FB5-1356 Add socket prioritisation mechanism for live streaming

### PR DESCRIPTION
**Pull request by cliffede**
Jira Link (use notation [PROJECT-123]): [FB5-1356]
Problem / Feature: See Jira
Solution: See Jira
Testing: See Jira
Jira or PR for parallel or older branches: <Enter link to Jira or another PR or delete this if not needed>
Concerns: None
Expert Reviewers: Any
Sections to be Reviewed: All
Expected maximum review time: 10 minutes

[FB5-1356]: https://ovationsystems.atlassian.net/browse/FB5-1356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: gst/rtsp-server/rtsp-media.h [hunk 1]
```diff
@@ -451,6 +458,12 @@ void                  gst_rtsp_media_set_rate_control (GstRTSPMedia * media, gbo
 GST_RTSP_SERVER_API
 gboolean              gst_rtsp_media_get_rate_control (GstRTSPMedia * media);
 
+GST_RTSP_SERVER_API
+GstElement *          gst_rtsp_media_get_pipeline (GstRTSPMedia * media);
```
Comment (vmbaillie) [inline gst/rtsp-server/rtsp-media.h]: is this relevant to this commit? this string doesn't appear elsewhere in the diff, see also get_bin below

Comment (cliffede) [reply gst/rtsp-server/rtsp-media.h]: Yes agreed, this should be removed

**Review by vmbaillie — APPROVED** [review]
